generated/beats/fields.ecs.yml

```diff
@@ -574,6 +574,12 @@
       description: 'IP address of the destination.
 
         Can be one or multiple IPv4 or IPv6 addresses.'
+    - name: ip.network_prefix
```

Comment:
just to know if this type of definition is possible:

ip is already a field of type IP not a json object

Reply:
Splitting out the ICMP type and code sounds like a good idea, would make querying a bit easier.

For the TCP flags, an array of flags sounds like a good approach, again to make querying easier.

Perhaps the IP address should become ip.address?

Reply:
honestly, "ip" fields exist all around since the first incarnation of the ECS. I would rather prefer to have a simple "network_prefix" field at the same level as the corresponding ip one